homeassistant/components/goodwe/sensor.py

```diff
@@ -121,6 +136,12 @@ class GoodweSensorEntityDescription(SensorEntityDescription):
         state_class=SensorStateClass.MEASUREMENT,
         native_unit_of_measurement=UnitOfFrequency.HERTZ,
     ),
+    "h": GoodweSensorEntityDescription(
+        key="h",
+        device_class=SensorDeviceClass.DURATION,
+        state_class=SensorStateClass.MEASUREMENT,
+        native_unit_of_measurement=UnitOfTime.HOURS,
+    ),
```

Comment:
What does this measurement entail? Is it an accumulation?

Reply:
Yes, it's description for sensors providing accumulated time values (e.g. hours PV produced energy daily/total)

Reply:
So it isn't a measurement, but a total? By your last response, this sounds like a total, not a measurement.

Reply:
Those values in hours are provided directly by the PV inverter and represent either daily or total durations of something (e.g. production of power). So the daily PV production in hours is in my understanding a measurement.
But again, similar to VA/var sensors above - these sensors are not new, they are already there since beginning, just until this change with the _attr_native_unit_of_measurement explicitly set to "h".

Reply:
> So the daily PV production in hours is in my understanding a measurement.

That is correct, but you also write "either daily or total durations".

> these sensors are not new, they are already there since beginning

As per above, that is not a reason to just say: "Oh ok, ignore the review comment".

Reply:
I am just explaining what/why is already there and that the meaning of this PR was to create non-breaking code cleanup.
What are you suggesting in relation to these "h" sensors ? I'm not sure I understand.

Reply:
> attr_native_unit_of_measurement explicitly set to "h".

Changing the unit of measurement, however, **IS** a breaking change. So, if that is the result of this PR in general, we should mark it as a breaking change and add a section to the PR description for the user so it can end up in the release notes.

Reply:
It is not changing the unit. So far it was hard-coded string "h" and now it will be UnitOfTime.HOURS, i.e. the very same value.
The only actual difference is that before the sensors had no device class and now will get SensorDeviceClass.DURATION.
But that is not breaking change, is it ?